### DISCUSSION TITLE
Fresh Code Base Runtime error

### DISCUSSION THREAD

**Discussion by stretch911** [PlummersSoftwareLLC/NightDriverStrip#222] [Q&A] [answered]
just pulled latest code base. compile was good and upload as well...however, seeing this for looks like 99 times

Pass 21 of 99: Connecting to Wifi SSID: eZone - ESP32 Free Memory: 45016, PSRAM:0, PSRAM Free: 0
E (106432) wifi:create wifi task: failed to create task
[105744][E][WiFiGeneric.cpp:680] wifiLowLevelInit(): esp_wifi_init 257
[105748][E][WiFiSTA.cpp:227] begin(): STA enable failed!

**Comment by davepl** [comment, marked as answer]
I’ve reworked the wifi (thought that’s not the cause), so you might want to pick up the changes I’ll check in later today.

My guess is low memory.  Decrease MAX_BUFFERS to 20, let me know if that fixes it!

- Dave

**Reply by stretch911** [reply]
i added that to "atomlight" in globals.h and it worked. thanks!!

**Comment by davepl** [comment]
You’re welcome!  Another way is to increase RESERVE_MEMORY by about 20K and leave MAX_BUFFERS uncapped.
